tools/niminst/makefile.tmpl

```diff
@@ -141,7 +135,16 @@ endif
 ifeq ($(ucpu),armv6l)
   mycpu = arm
 endif
-ifndef ucpu
+ifeq ($(ucpu),armv7l)
```

Comment:
General observation:

It would be great if we could add the supported CPUs to niminst instead of repeating them in bash and the makefile.

Reply:
I don't see how this could work, platfrom detection needs to be procedural code, a "list of CPUs" is not gonna cut it.

Reply:
A mapping then? That's all this seems to be.

Reply:
Better remove the makefile then. It's not officially supported anyway.

Reply:
I'm good with removing

Reply:
I will remove it the next time this comes up.